tests/ui/feature-gates/feature-gate-type_alias_impl_trait.rs

```diff
@@ -12,7 +12,10 @@ fn define() -> Bar {
 
 type Foo2 = impl Debug;
 
-fn define2() {
+fn define2()
+where
+    Foo2: Debug,
```

Comment:
I think this should be implied from the TAIT? Because it's `impl Debug`?

Reply:
oh it is implied, but the TAIT needs to be in the signature ^^ So I made a trivial bound. Could also use `Foo2: 'static` or potentially even `Foo2:,`

Reply:
Ohhh. Okay :)